### ISSUE TITLE
publish latest to npm

### ISSUE THREAD

**Issue by the-simian** [andyearnshaw/Intl.js#251]
I saw this update: https://github.com/andyearnshaw/Intl.js/pull/234
It looks like this hasn't been published to npm. can we get an incremented minor version
1.2.5 -> 1.2.6 and get this published? would help a ton! thanks!

**Comment by ptomasroos** [comment]
Same here, waiting for it.

**Comment by caridy** [comment]
working on it :)

**Comment by the-simian** [comment]
howdy! its been a bout a month. I'd love to see this get out on npm :)

**Comment by swernerx** [comment]
If you need specific help in getting this done it would be very appreciated.

**Comment by caridy** [comment]
I'm at tc39 meeting this week, will try to get the package out there but it is always challenging. I need help testing master in all browsers that we are trying to support, because sauce-lab is not really working at the moment.

**Comment by the-simian** [comment]
I am pointing directly to the Master branch and I can confirm its working as advertised in react-native environments, so I can give you a thumbs up for android/ios via RN.

**Comment by the-simian** [comment]
@caridy I believe this is on npm now.

**Comment by alexwalter88** [comment]
I don't think so. The version published to npm is still 1.2.5

**Comment by the-simian** [comment]
@alexwalter88 , you're right!

**Comment by eliseumds** [comment]
Any updates?

**Comment by the-simian** [comment]
Hello, its been over half a year. can you please type `npm publish` ?

**Comment by swernerx** [comment]
You all might want to try this fork/alternative: https://www.npmjs.com/package/lean-intl

**Comment by JakeChampion** [comment]
@caridy Would running this against all the browsers that the polyfill-service target for this polyfill help to get it published?

**Comment by JakeChampion** [comment]
@caridy Here is a PR which is passing on Travic CI -- https://github.com/andyearnshaw/Intl.js/pull/288

**Comment by anescobar1991** [comment]
@caridy with @JakeChampion's passing build is there anything else you need that we can help with before publishing?

**Comment by the-simian** [comment]
@caridy I don't think this will be published before 2018, can we expect this to be published before 2019? Asking for a friend that's been waiting over a year for you to type `npm publish` in a console. Thanks.
